### ISSUE TITLE
It would be extremely helpful if there were examples of the configuration files

### ISSUE THREAD

**Issue by jzhanglsw** [glisignoli/fcreplay#167]
The readme specifies these files:
Configuration file: config.json
Appened description: description_append.txt
Google api client secrets: .client_secrets.json
Youtube upload credentials: .youtube-upload-credentials.json
Archive.org secrets: .ia

But I don't see any clues as to how to generate them? It seems like I'm supposed to make a config.json file but I have no idea what's supposed to go inside it.
Seems like description_append.txt is just whatever I want in the description of the video
.client_secrets.json is from google that much I understand
.youtube-upload-credentials.json is from a different repo but they don't show how to generate that file in their readme either, I don't understand how that differs from .client_secrets.json
Lastly, I don't know what's supposed to go in .ia

I'm a software engineer so I'm either extremely dense or there's information that needs to be given here.

**Comment by glisignoli** [comment]
Hi!

Sorry yeah, my documentation wasn't really written thinking anyone else would be using this project but some help:

An example of a 'working' config: https://github.com/glisignoli/fcreplay/blob/master/fcreplay/tests/common/config_good.json

The following files are needed if you want to upload to youtube automatically. 
.client_secret.json
.youtube-upload-credentials.json
I think this repository might help explain how to get them: https://github.com/tokland/youtube-upload

The .ia file is api credentials for archive.org

Try just giving some emtpy json files for .client_secret.json, .youtube-upload-credentials.json and .ia. It 'might' work without them

The description_append.txt file can be an empty txt file. It's used to append a static description to whatever videos are recorded.

**Comment by jzhanglsw** [comment]
No worries about the documentation. I think it's awesome you made this and
it solves a real problem many people are having in the fighting game
community who are using fightcade to play games. I'll give your comments a
try.

Who knows, maybe I'll even fork the repo and add the NoSql database you
wanted. :)

**Comment by jzhanglsw** [comment]
Hey, I'm really close to getting the repo running. But I'm getting one last
error here:

λ docker-compose up
Creating network "fcreplay-master_postgres" with the default driver
Creating network "fcreplay-master_world" with the default driver
Pulling fcreplay-tasker-check_top_weekly
(fcreplay/image:latest)...ERROR: The image for the service you're
trying to recreate has been removed. If you continue, volume data
could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling fcreplay-tasker-check_top_weekly
(fcreplay/image:latest)...ERROR: pull access denied for
fcreplay/image, repository does not exist or may require 'docker
login': denied: requested access to the resource is denied

Is the docker image still available?

**Comment by glisignoli** [comment]
Ah you need to build the images manually with docker compose.

Try: docker-compose build

**Comment by glisignoli** [comment]
I'll spend some time in the next couple of days to try and update the documentation.

**Comment by g4-studios** [comment]
Hey, thanks for answering so quickly. I've been doing more research and it seems I was able to build the image, but when running the `docker-compose up` command, it seems like it works for a while until it throws this error:

> Error response from daemon: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:75: mounting "/path/to/config.json" to rootfs at "/root/config.json" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type

if anything, I put all the required json and txt files in the fcreplay-master folder.

**Comment by glisignoli** [comment]
Yeah. I'm working on some documentation about this now. It's needed some work anyway. Hopefully I'll have a new commit today with a much better guide on how to get it up and running

**Comment by glisignoli** [comment]
@g4-studios, @jzhanglsw:

I've push to master some changes and updated instructions. Delete whatever config.json and docker-compose.override.yml files you have follow the README.md

**Comment by jzhanglsw** [comment]
Okay will do, I'll try and test this out this week.

**Comment by g4-studios** [comment]
I couldn't setup pipenv, not even with the other instructions that were in pipenv's own website :(

**Comment by glisignoli** [comment]
> I couldn't setup pipenv, not even with the other instructions that were in pipenv's own website :(

What did you try? Are you on windows/mac/linux?

**Comment by g4-studios** [comment]
I'm on windows. The command simply doesn't work, despite having pipenv already installed by the command that's on their website.

![image](https://user-images.githubusercontent.com/60892378/172708933-ec2efd88-b4ab-4022-be8b-e8159e25486b.png)

![image](https://user-images.githubusercontent.com/60892378/172708966-9f7de046-e88d-4ca5-9c8f-21ad8638f473.png)

**Comment by glisignoli** [comment]
Looks like pipenv isn't in your path. Otherwise it would have found it. That isn't a problem with fcreplay though.

I'm going to recommend that you do this under WSL or Linux. This project hasn't been tested on windows (and never will be).

**Comment by g4-studios** [comment]
It is
![image](https://user-images.githubusercontent.com/60892378/172724196-5d441787-0b2f-4931-9c31-3997284b7890.png)

Alright, I'll try WSL and see how it goes.

**Comment by g4-studios** [comment]
I've been running into these errors for hours, even with root.

![image](https://user-images.githubusercontent.com/60892378/172772745-63e38aa6-6b38-4441-ae39-ca75f44852b4.png)
![image](https://user-images.githubusercontent.com/60892378/172772778-64f17cc2-ed32-47c2-a6ad-9dc234b93156.png)

As shown, pip is up to the latest version and also using Python 3.10. I looked for the `cannot import name 'html5lib' from 'pip._vendor'` error but the supposed solution was to update pip which is already at its latest version. I'm pretty sure this is easily fixable as I am not savvy in these stuff but stackoverflow is no longer helping me lol
